### ISSUE TITLE
pyats logs view | --port argumnet is not working

### ISSUE THREAD

**Issue by urskog84** [CiscoTestAutomation/pyats#30]
Can not user argumnet  --port
clarifying.. this is from inside the pyats docker contaner
```
# pyats logs view --host localhost --port 8080 -v
or 
# pyats logs view --port 8080 -v
```

error
```
Logfile: /root/.pyats/archive/19-Dec/test_job.2019Dec06_12:50:49.819093.zip
Traceback (most recent call last):
  File "src/pyats/cli/base.py", line 202, in pyats.cli.base.Command.main
  File "src/pyats/cli/base.py", line 332, in pyats.cli.base.CommandWithSubcommands.run
  File "src/pyats/log/commands/logs.py", line 136, in pyats.log.commands.logs.ViewSubcommand.run
  File "src/pyats/log/commands/logs.py", line 157, in pyats.log.commands.logs.ViewSubcommand.run
UnboundLocalError: local variable 'displayed_port' referenced before assignment


local variable 'displayed_port' referenced before assignment

```

**Comment by bastell** [comment]
Hi,
This has been fixed in our upcoming release v19.12 on Dec 17, in just a few days. Please verify once it's available.

**Comment by urskog84** [comment]
One step in the right direction...

```
root@2f704cf747a1:/pyats# pyats logs view --port 80 -v
Logfile: /root/.pyats/archive/20-Jan/test_job.2020Jan16_21:04:29.276865.zip

View at: 
    http://localhost:8080/result

Press Ctrl-C to exit

--------------------------------------------------------------------------------

Cannot automatically start your browser: X-forwarding is not enabled for your session. 
Navigate to the above url manually in your preferred browser.
Using selector: EpollSelector
```
but i can not display the logs in the web browser.

i start the docker container with this command
```
docker run --env-file .env -p 8080:8080 -it $(docker build . -q) /bin/bash
```

**Comment by bastell** [comment]
Hi,
Try setting the host to 0.0.0.0 when running pyats log viewer to allow for external connections
`pyats logs view --host 0.0.0.0 --port 8080 -v`

**Comment by simingy** [comment]
this issue should be resolved in v20.1

**Comment by muhammad-rafi** [comment]
Sorry for commenting on the closed issue but I found the same issue when I created a container but open up the port, I wanted to see logs on via my host machine. 

I ran this command to run a container 

```
$ docker run -itd --hostname pyats-devnet --name pyats-devnet \
    -v "$(pwd)":/root \
    -p 48000:48000 \
    ciscotestautomation/pyats:latest
```

I tried to run the log viewer like this 
```
(pyats) root@pyats-devnet:/pyats# pyats logs view --host 0.0.0.0 --port 48000 --no-browser --latest -vvv
Logfile: /root/.pyats/archive/22-Oct/basic_example_job.2022Oct09_11:54:18.652364.zip

View at: 
    http://pyats-devnet:48000/

Press Ctrl-C to exit

--------------------------------------------------------------------------------

Using selector: EpollSelector

```

and when I try to access this url on my local system via  http://localhost:48000/, I got `This site can’t be reached` and even curl shows connection refused. 

```
$ curl http://localhost:48000/
curl: (7) Failed to connect to localhost port 48000 after 5 ms: Connection refused
``` 

I also tried this as well but no difference 
`pyats logs view --host 127.0.0.1 --port 48000 --no-browser --latest -vvv`

Please let me know if I am doing something wrong ?
